tests/features/collection/homepage.feature

```diff
@@ -6,11 +6,11 @@ Feature: Collection homepage
 
   Background:
     Given users:
-      | Username |
-      | Frodo    |
-      | Boromir  |
-      | Legoloas |
-      | Gimli    |
+      | Username | Status |
+      | Frodo    | 1      |
```

Comment:
I would love to see human readable values here, like `active` and `blocked` instead of these weird numbers. The product owner and functional contractors don't know what it means if a status is set to `1`.

Reply:
Yes I looked into this and saw that we didn't do this yet in any test that uses the Status column. I didn't want to reject for this, but we can take the chance now that we have to fix some other remarks.

Reply:
Yes it got into my mind twice but I didn't want to reject only for this, as I saw that sadly we used this syntax in other places too. But since now the ticket needs more work, let's take the chance to fix it.